docker/post-build.sh

```diff
@@ -48,7 +48,7 @@ bitbake -S none ${_images}
 # Check linux-intel specifically in addition to images, because it did
 # rebuild at some point and even though bitbake-diffsigs should
 # recurse to it, that's not guaranteed to work.
-for target in linux-intel ${_images}; do
+for target in intel-linux ${_images}; do
```

Comment:
Kernel package is called linux-intel, we also support linux-intel-rt as well as linux-yocto and linux-yocto-rt, the same ones supported by meta-intel BSP:
http://git.yoctoproject.org/cgit/cgit.cgi/meta-intel/tree/common/recipes-kernel/linux